packages/gestalt/src/InfoButton.jsdom.test.js

```diff
@@ -0,0 +1,148 @@
+// @flow strict
+import { screen, render, act } from '@testing-library/react';
+import InfoButton from './InfoButton.js';
+
+describe('InfoButton', () => {
```

Comment:
These tests are great! Revisit them after making the suggested API changes to the component, as it looks like some will change (or can be removed)